src/clj/game/cards/programs.clj

```diff
@@ -1760,18 +1774,52 @@
       (strength-pump (->c :credit 2 {:stealth 1}) 4 :end-of-run)]}))
 
 (defcard "Hush"
-  ;; TODO - come back to this once disabling cards has been reworded. nbkelly, jan 2023
-  {:implementation "Unimplemented - corp must use /disable-card and /enable-card on ice to fix any issues (ie anansi, afshar, magnet)"
-   :hosting {:card #(and (ice? %)
-                         (can-host? %))}
+  (let [reset-card-to-printed-subs
+        (fn [state side card]
+          (let [card (get-card state card)
+                subs (vec (remove #(or (:variable %)
+                                       (not (:printed %)))
+                                  (:subroutines card)))
+                ;; special case for hive, because it unprints subroutines...
+                subs (if (= "Hive" (:title card))
+                       (subroutines-init card (card-def card))
+                       subs)
+                new-card (assoc card :subroutines subs)]
+            (update! state :corp new-card)
+            (trigger-event state side :subroutines-changed (get-card state new-card))))
```

Comment:
If you're worried about loops, only call this when the old card `:subroutines` don't match the new card `:subroutines`. That way, you won't get into loops when the same card triggers over and over again.

Reply:
I *think* it's not possible - so long as we don't ever make effects like warden fatuma actually work as written, and constantly apply an ability.
In that case, I think your example would cause the same problem.
I realized I can just refactor this part to card-init the subs though, so I did that.